index.html

```diff
@@ -0,0 +1,867 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Positioning and floating elements</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lora:wght@400;500;700&family=Poppins:wght@400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+  <body>
+    <nav class="d-flex">
+      <div class="nav-item d-flex">
+        <a href="#" class="d-flex">
+          <img class="nav-icons" src="./assets/menu.svg" alt="menu-icon" />
+        </a>
+        <a href="#" class="d-flex">
+          <img class="nav-icons" src="./assets/search.svg" alt="search-icon" />
+        </a>
+        <a href="#" class="nav-link">
+          Space & cosmos
+        </a>
+      </div>
+      <div class="logo-container">
+        <img class="nav-item logo" src="./assets/NYTLogo.svg" alt="NYT-logo" />
+      </div>
```

Comment:
The logo should be a link. You can do this by adding an anchor tag

Reply:
As requested by the previous TSE, please make the logo a link by adding an anchor tag